sdk/communication/azure-communication-phonenumbers/setup.py

```diff
@@ -66,7 +66,7 @@
     python_requires=">=3.6",
     install_requires=[
         "msrest>=0.6.21",
-        'azure-core<2.0.0,>=1.20.0',
+        'azure-core<2.0.0,>=1.23.0',
```

Comment:
This was generated automatically?

Reply:
nope, it is a manual change because of the dependencies conflict(in the azure-mgmt-core the core version was upgraded)